### PR TITLE
Fix artifact upload

### PR DESCRIPTION
**Pull request by bkurtz**
## Description
#2585 refactored the split between regular/base images and introduced an issue with artifact upload naming.

## Type of change
- [ ] New feature (backwards compatible)
- [ ] New feature (breaking API changes)
- [x] Bug fix
- [ ] Other (Documentation, CI, ...)

## [Contribution checklist](https://www.cvxpy.org/contributing/index.html#contribution-checklist)
- [x] Add our license to new files.
- [x] Check that your code adheres to our coding style.
- [ ] Write unittests.
- [x] Run the unittests and check that they’re passing.
- [x] Run the benchmarks to make sure your change doesn’t introduce a regression.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -172,5 +172,5 @@ jobs:
         if: ${{github.event_name == 'push' && env.USE_OPENMP != 'True'}}
         uses: actions/upload-artifact@v4
         with:
-          name: wheels-${CVXPY_BASE_SUFFIX}${{ matrix.os }}-${{ matrix.python-version }}
+          name: wheels-${{ env.CVXPY_BASE_SUFFIX }}${{ matrix.os }}-${{ matrix.python-version }}
```
Comment (bkurtz) [inline .github/workflows/build.yml:175]: oops! I was assuming shell-style interpolation would work here, but it doesn't because we're not in a shell!

**Comment by Transurgeon** [comment]
@bkurtz nice catch! I was just going to open an issue related to this. However, there still seems to be a failing CI for python3.11 on ubuntu (something to do with repairing wheels?). It would be nice if you could fix that

**Comment by bkurtz** [comment]
Okay, based on builds on 3699b804c067af160dbe45bcab63a3c37a783c44, I think this should be good to go?

**Review by Transurgeon — APPROVED** [review]
LGTM. Thanks for the fast fix @bkurtz .

**Review by SteveDiamond — APPROVED** [review]

**Pull request merged**